src/pool/cli.c

```diff
@@ -1326,6 +1326,12 @@ dc_pool_query(tse_task_t *task)
 	return rc;
 }
 
+int
+dc_pool_query_target(tse_task_t *task)
```

Comment:
I was thinking we will implement this 'query target' placeholder instead of changing current pool query. And to keep implementation simple, I think query single target should be enough. (Of course, the wire format could be defined to be able to support multiple targets for later potential extending)

Reply:
will implement daos_pool_query_target API in a different patch

Reply:
If per target query is implemented in this way, I don't think the daos_pool_query_target() is necessary anymore, we can remove the placeholder later.